src/main/java/com/example/booksstore/controller/BookController.java

```diff
@@ -5,16 +5,18 @@
 import com.example.booksstore.service.BookService;
 import java.util.List;
 import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RequiredArgsConstructor
 @RestController
-@RequestMapping (value = "/books")
+@RequestMapping(value = "/books")
```

Comment:
actually, it should be `/api/books` as per the web HW I think, or you can add a common path prefix for each endpoint in your application.properties